src/components/CreditCard.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+
+function CreditCard(props) {
+  return (
+     <div>
+    <p>{props.type}</p>
+    <p>{props.number}</p>
+    <p>{props.expirationMonth}</p>
+    <p>{props.expirationYear}</p>
+    <p>{props.bank}</p>
+    <p>{props.owner}</p>
+    <p>{props.bgColor}</p>
```

Comment:
The idea was to pass the bg colour and colour on style. To do that, you could've followed the logic of the Box Color iteration.

`<div style={{backgroundColor: bgColor, color: color}}>`